README.md

```diff
@@ -22,36 +22,24 @@ In general, application-specific code will live outside of this package.
 
 ## Environment
 
-Platform configuration is read from environment variables, which must be
-declared in `.mupweb.config` in your home directory (i.e.
-`$HOME/.mupweb.config`) in the following format:
+The platform code uses [node-convict](https://github.com/mozilla/node-convict)
+for configuration management in `src/util/config.js`. All
+development defaults are set within `config.js` but can be overwritten 
+with a root directory `config.dev.json` that is gitignored.
 
-```
-API_HOST=api.meetup.com
-API_PROTOCOL=https
-DEV_SERVER_PORT=8000
-ASSET_SERVER_HOST=0.0.0.0
-ASSET_SERVER_PORT=8001
-OAUTH_AUTH_URL=https://secure.meetup.com/oauth2/authorize
-OAUTH_ACCESS_URL=https://secure.meetup.com/oauth2/access
-MUPWEB_OAUTH_KEY=<check with an admin>
-MUPWEB_OAUTH_SECRET=<check with an admin>
-PHOTO_SCALER_SALT='<check with admin>'  # single quotes are required
-CSRF_SECRET='<any random string over 32 characters long>'
-COOKIE_ENCRYPT_SECRET='<any random string over 32 characters long>'
-```
-
-**Note**: you _can_ use `dev.meetup.com` URLs for `API_HOST`, `OAUTH_AUTH_URL`,
-and `OAUTH_ACCESS_URL`, but you will need to ensure that your devbox is up and
-running with a recent build of Meetup classic.
-
-To automatically add these env variables into your terminal session,
-`source` the config file in your `.bashrc` or `.zshrc`:
+For example, if you want to use the production api in dev, 
+create the following `config.dev.json` file:
```

Comment:
Actually I should have made the instructions comment here - we need to document what a 'full' configuration file looks like, ideally with explanations of each key, but that can come later. The main thing that people will need is a reference for what a complete config looks like and I think you can just link/point them to `config.test.json` here